.circleci/continue_config.yml

```
@@ -0,0 +1,284 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.4.0
+
+# the default pipeline parameters, which will be updated according to
+# the results of the path-filtering orb
+parameters:
+  run-build-kedro-telemetry:
+    type: boolean
+    default: false
+  run-build-kedro-docker:
+    type: boolean
+    default: false
+  run-build-kedro-airflow:
+    type: boolean
+    default: false
+
+
+commands:
+  setup_conda:
+    parameters:
+      python_version:
+        type: string
+    steps:
+      - run:
+          name: Create virtual env
+          command: |
+            # Get rid of pyenv stuff
+            sudo rm -rf .pyenv/ /opt/circleci/.pyenv/
+            # Download and install miniconda
+            curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  > miniconda.sh
+            bash miniconda.sh -b -p $HOME/miniconda
+            # Create an anaconda virtualenv for python_version and make that the default python interpreter
+            echo ". /home/circleci/miniconda/etc/profile.d/conda.sh" >> $BASH_ENV
+            echo "conda deactivate; conda activate kedro_plugins" >> $BASH_ENV
+            . /home/circleci/miniconda/etc/profile.d/conda.sh
+            conda create --name kedro_plugins python=<<parameters.python_version>> -y
+            source $BASH_ENV
+
+  setup_requirements:
+    parameters:
+      plugin:
+        type: string
+    steps:
+      - run:
+          name: Install pip setuptools
+          command: |
+            make install-pip-setuptools
+      - run:
+          name: Install kedro and test requirements
+          command: |
+            cd <<parameters.plugin>>
+            pip install git+https://github.com/kedro-org/kedro
```

Maybe just to be explicit?
```suggestion
            pip install git+https://github.com/kedro-org/kedro@main
```